File: app/src/main/java/com/blog/demo/custom/widget/GroupTwoView.java
package com.blog.demo.custom.widget;

import android.content.Context;
import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.widget.LinearLayout;
import android.widget.TextView;

import com.blog.demo.R;

public class GroupTwoView extends LinearLayout {

    public GroupTwoView(Context context) {
        super(context);
    }

    public GroupTwoView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    protected void onFinishInflate() {
        super.onFinishInflate();

        TextView tv = findViewById(R.id.text_view);
        tv.setText("This is group two.");
    }

}

File: app/src/main/java/com/blog/demo/custom/widget/GroupTwoView.kt
package com.blog.demo.custom.widget

import android.content.Context
import android.util.AttributeSet
import android.widget.LinearLayout
import android.widget.TextView
import com.blog.demo.R

class GroupTwoView(context: Context, attrs: AttributeSet?, defStyleAttr: Int) :
    LinearLayout(context, attrs, defStyleAttr) {

    constructor(context: Context) : this(context, null)

    constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)

    override fun onFinishInflate() {
        super.onFinishInflate()
        val tv: TextView = findViewById(R.id.text_view)
        tv.text = "This is group two."
    }

}